src/viewer/viewer.ts

```diff
@@ -1032,74 +1025,78 @@ export default class Viewer {
     this.orthographicCamera.zoom = this.height / height
   }
 
+  /**
+   * Convert an absolute clip value to a relative one using bRadius.
+   *
+   * 0.0 -> 50.0
+   * bRadius -> 0.0
+   */
+  absoluteToRelative (d: number) :number {
+    return 50 * (1 - d / this.bRadius)
+  }
+
+  /**
+   * Convert a relative clip value to an absolute one using bRadius
+   *
+   * 0.0 -> bRadius
+   * 50.0 -> 0.0
+   */
+  relativeToAbsolute (d: number) : number {
+    return this.bRadius * (1 - d/50)
+  }
+
+  /**
+   * Intepret clipMode, clipScale and set the camera and fog clipping.
+   * Also ensures bRadius and cDist are valid
+   */
   private __updateClipping () {
     const p = this.parameters
 
-    const debugClipping = false
+    // bRadius must always be updated for material-based clipping
+    // and for focus calculations
+    this.bRadius = Math.max(10, this.boundingBoxLength * 0.5)
 
-    // clipping
+    // FL: Removed below, but leaving commented as I don't understand intention
+    // this.bRadius += this.boundingBox.getCenter(this.distVector).length()
```

Comment:
I don't know about this enlarging the bbox either -- I did a bit of git archaeology; it blames back at least to 600dc160b (@arose 2016-06-22, "refactored/cleaned-up camera/view controls"):
```diff
-        bRadius += boundingBox.center( distVector )
-            .add( rotationGroup.position )
-            .length();
+        bRadius += boundingBox.center( distVector ).length();
```

Reply:
Hmm, perhaps Alex remembers? @arose